go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/golang-migrate/migrate/v4 v4.2.5
 	github.com/gomodule/redigo v2.0.0+incompatible // indirect
 	github.com/google/go-querystring v1.0.0
-	github.com/google/uuid v1.0.0
+	github.com/google/uuid v1.1.1
```

Comment:
Are this changes intended?

Reply:
no, it happened automatically. Afaik we decided to just let go.mod and go.sum be updated when it happens. Maybe ask @j4k4 if you think this is the wrong behaviour.

Reply:
Do we have defined a general rule here?
But minor updates should be safe, right?